src/main.cpp

```diff
@@ -3125,7 +3126,7 @@ bool CheckBlock(const CBlock& block, CValidationState& state, bool fCheckPOW, bo
     {
         nSigOps += GetLegacySigOpCount(tx);
     }
-    if (nSigOps > MAX_BLOCK_SIGOPS)
+    if (nSigOps > Policy::blockSigOpAcceptLimit(blockSize))
```

Comment:
According the the BUIP 40 document and my reading of the BU code, they are indeed using the `GetLegacySigOpCount` function for determining the sigop limits.